CHANGES.md

```diff
@@ -1,3 +1,7 @@
+## [0.2.1] (2018-11-27)
+
+- Fix `tmkms yubihsm keys import` command (#113)
+
```

Comment:
```suggestion
- Encode node (and softsign) private keys as Base64 (#127)

```

Reply:
Added this, although with some slightly different formatting due to other changes